app/components/DropDown/DropDownStyles.ts

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native';
+import {
+  Colors,
+  horizontalScale,
+  moderateScale,
+  verticalScale,
+} from '../../theme';
+
+export const styles = StyleSheet.create({
+  dropDownContainer: {
+    position: 'absolute',
+    right: 0,
+    backgroundColor: Colors.lightGreen,
+    borderRadius: 20,
```

Comment:
used theme scale functions

Reply:
We are instructed that when using border radius don't use any scale functions , and also when right/left is defined to 0 no need to use scale functions